library/std/src/f32.rs

```diff
@@ -876,4 +876,32 @@ impl f32 {
     pub fn atanh(self) -> f32 {
         0.5 * ((2.0 * self) / (1.0 - self)).ln_1p()
     }
+
+    /// Linear interpolation between `start` and `end`.
+    ///
+    /// This enables the calculation of a "smooth" transition between `start` and `end`,
```

Comment:
The word "smooth" might suggest some kind of non-linear curve (e.g. a [smooth step](https://www.google.com/search?q=smooth+step&tbm=isch) function). Maybe `a linear transition` would be better here.